<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QWidget" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/img/retouch_2025011309461031.png</normaloff>:/img/retouch_2025011309461031.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>331</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入账号...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>331</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入密码...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>551</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L O G I N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color:white;
	background-color: rgba(168, 90, 78, 190);
border-radius:20px;
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>550</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color:white;
	background-color: rgba(98, 121, 153, 190);
	border-radius:20px;
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>750</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/926386dea79cf2ea36dcb89162ddac4.jpg);</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>usernameEdit</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label</zorder>
  <zorder>loginButton</zorder>
  <zorder>registerButton</zorder>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
